### PR TITLE
[AutoML] Clone AutoML feature branch into docs/samples/automl

### PR DESCRIPTION
**Pull request by daholste**

### CODE REVIEW

**Comment by daholste** [comment]
@natke to review if you have a moment --
Thank you for all the help!!!!

File: .openpublishing.publish.config.json [hunk 1]
```diff
@@ -50,6 +50,12 @@
       "url": "https://github.com/dotnet/machinelearning",
       "branch": "master",
       "branch_mapping": {}
+    },
+    {
+      "path_to_root": "docs/samples_automl",
+      "url": "https://github.com/dotnet/machinelearning",
```
Comment (sfilipi) [inline .openpublishing.publish.config.json]: you'd want to put the automl repo url here

File: .openpublishing.publish.config.json [hunk 2]
```diff
@@ -50,6 +50,12 @@
       "url": "https://github.com/dotnet/machinelearning",
       "branch": "master",
       "branch_mapping": {}
+    },
+    {
+      "path_to_root": "docs/samples_automl",
```
Comment (sfilipi) [inline .openpublishing.publish.config.json]: You'd need to create this folder, it is not there atm.

**Comment by sfilipi** [comment]
@shauheen to check on whether we are ok with the shared infrastructure.

**Comment by shmoradims** [comment]
Need to confirm this doesn't create any coupling between ML.NET and AutoML document generation, document index, and monikers.

**Pull request closed without merging**